dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -264,14 +266,27 @@ winrt::guid Microsoft::Windows::AppNotifications::Helpers::RegisterComActivatorG
         THROW_IF_FAILED(hr);
     }
 
-    std::wstring notificationAppId{ Microsoft::Windows::AppNotifications::Helpers::RetrieveNotificationAppId() };
-    RegisterAssets(notificationAppId, registeredGuid);
+    return registeredGuid;
+}
 
-    // Remove braces around the guid string
-    return winrt::guid(registeredGuid.substr(1, registeredGuid.size() - 2));
+// Try the following techniques to retrieve display name and icon:
+// 1. Based on the best app shortcut, using the FrameworkUdk.
+// 2. From the current process.
+// 3. Set a default DisplayName, but leave empty the icon file path so Shell can set a default icon.
+AppNotificationAssets Microsoft::Windows::AppNotifications::Helpers::GetAssetsHelper()
+{
+    AppNotificationAssets assets{};
+
+    if (FAILED(Microsoft::Windows::AppNotifications::ShellLocalization::RetrieveAssetsFromShortcut(assets)) &&
```

Comment:
Can you please use a namespace for ShellLocalization? These lines are unnecessarily long.